CY-3499 - Add Getting Started section to UI documentation (#1416)
cloudify-cosmo/docs.getcloudify.org: content/working_with/console/getting-started/navigation.md

## Top header

Thick bar at the top of the screen presents main information about the product and global navigation menus.

### Left side

The following elements are presented in the left side of the top header bar:
 
* **Logo**
* **Product name**
* **License edition** - visible in non-community versions
* **Product version** - only major an minor digits are presented
* **License status** - on click you are redirected to License Management page. See [License Management]({{< relref "install_maintain/installation/manager-license.md" >}}) page for more details. 

{{% note %}}
You can customize it using white-labelling feature described in [User Configuration]({{< relref "working_with/console/customization/user-configuration.md" >}}).
{{% /note %}}

### Right side

The following elements are presented in the right side of the top header bar:

* **System status** - on hovering the status you can see more detailed information about system services
* **Tenant menu** - click on the current tenant name to open dropdown selection menu and change the current tenant  
* **Help menu** - click on the ![Help icon]( /images/ui/icons/help-icon.png ) to open menu with links to support pages 

> * **User menu** - click on the username to open [Edit Mode]({{< relref "working_with/console/customization/edit-mode.md" >}}), [Template Management]({{< relref "working_with/console/customization/templates-mgmt.md" >}}), [License Management]({{< relref "install_maintain/installation/manager-license.md" >}}), change password or log out

Reviewer: ```suggestion
* **User menu** - click on the username to enter [Edit Mode]({{< relref "working_with/console/customization/edit-mode.md" >}}), open [Template Management]({{< relref "working_with/console/customization/templates-mgmt.md" >}}) or [License Management]({{< relref "install_maintain/installation/manager-license.md" >}}), change password or log out
```

{{% note %}}

> Set of the available options can differ between admin and non-admin users. The list presents all the options available for admins.

Reviewer: ```suggestion
Set of the available options can differ between admin and non-admin users. The list above presents all the options available for admins.
```

{{% /note %}}

## Left sidebar

In the left side of the screen the **Pages list** is presented. Click on the name of the page to change the view.

{{% note %}}

> Set of the available options can differ between admin and non-admin users. The list presents all the pages available for admins.

Reviewer: What list?

{{% /note %}}

> Detailed description about pages content can be found at [Pages]({{< relref "working_with/console/pages/_index.md" >}}) page.

Reviewer: ```suggestion
Detailed description of pages content can be found at [Pages]({{< relref "working_with/console/pages/_index.md" >}}) page.
```

In the left side of the screen the **Pages list** is presented. Click on the name of the page to change the view.

{{% note %}}

> Set of the available options can differ between admin and non-admin users.

Reviewer: ```suggestion
List of available pages can differ between admin and non-admin users.
```
